### PR TITLE
Fixing Katas references to indicate the correct Visual Studio version

### PR DESCRIPTION
**Pull request by ricardo-espinoza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ QuantumKatas/
   BasicGates/
     README.md                  # Instructions specific to this kata.
     .vscode/                   # Metadata used by Visual Studio Code.
-    BasicGates.sln             # Visual Studio 2019 solution file.
+    BasicGates.sln             # Visual Studio 2022 solution file.
```
Comment (tcNickolas) [inline README.md:215]: You can remove version here, since technically it's still a VS 2019 project

Comment (tcNickolas) [inline README.md:215]: You can remove version here, since technically it's still a VS 2019 project

**Review by tcNickolas — CHANGES_REQUESTED** [review]
Please update the readme to include complete instructions for setting up environment for .NET6 that are not covered in our documentation:

* The link at "[install guide for the Quantum Development Kit]" should point to https://docs.microsoft.com/en-us/azure/quantum/install-command-line-qdk
* Add a new section under "Quantum Development Kit Installation" section explaining the update to .NET6, installing it manually if not using VS, where to get the extensions and how to install them (including the bit about uninstalling old VS Code extension before installing the new one), installing the beta version of IQ# with the specific version manually, and any other details that are different from our official installation instructions

File: README.md (hunk 1 above)
Comment (ricardo-espinoza) [reply README.md:215]: Sounds good!

**Review by tcNickolas — APPROVED** [review]
Looks reasonable, this should tide us over to the next week's release.

**Pull request merged**